baklava/package.json

```diff
@@ -129,4 +129,4 @@
         "./icons/**/*",
         "./resources/**/*"
     ]
-}
+}
```

Comment:
0/10 PR, No newline

```suggestion
}

```

Reply:
#GoldyLovesEmNewlines

Reply:
idk why but my editor removes newlines

Reply:
You can turn off all formatting on save in VSCode

Reply:
but i want auto format, i just dont want this